pokemongo_bot/cell_workers/update_title_stats.py

```diff
@@ -152,6 +155,7 @@ def _process_config(self):
         """
         self.min_interval = int(self.config.get('min_interval', self.DEFAULT_MIN_INTERVAL))
         self.displayed_stats = self.config.get('stats', self.DEFAULT_DISPLAYED_STATS)
+        self.terminal = self.config.get('terminal', self.TERMINAL)
 
```

Comment:
Here you rewrite all the configs again. IMHO, this code should  go to `__init__.py` and instead of the constants as default we can have actual values of these constants.